pkg/vulnsrc/ubuntu/ubuntu_test.go

```diff
@@ -60,3 +62,65 @@ func TestVulnSrc_Update(t *testing.T) {
 		})
 	}
 }
+
+func TestUbuntuStatusFromStatus(t *testing.T) {
+	tests := []struct {
+		name     string
+		status   string
+		expected types.Status
+	}{
+		{
+			name:     "not-affected",
+			status:   "not-affected",
+			expected: types.StatusWillNotFix,
+		},
+		{
+			name:     "DNE",
+			status:   "DNE",
+			expected: types.StatusWillNotFix,
+		},
+		{
+			name:     "not-vulnerable",
+			status:   "not-vulnerable",
+			expected: types.StatusWillNotFix,
+		},
+		{
+			name:     "ignored",
+			status:   "ignored",
+			expected: types.StatusWillNotFix,
+		},
+
+		{
+			name:     "deffered",
+			status:   "deffered",
+			expected: types.StatusFixDeferred,
+		},
+		{
+			name:     "needed",
+			status:   "needed",
+			expected: types.StatusPending,
+		},
+		{
+			name:     "pending",
+			status:   "pending",
+			expected: types.StatusPending,
+		},
```

Comment:
```suggestion
			name:     "needed",
			status:   "needed",
			expected: types.StatusFixDeferred,
		},
		{
			name:     "pending",
			status:   "pending",
			expected: types.StatusFixDeferred,
		},
```
In line with above comments